src/Converter/Feature2Mesh.js

```diff
@@ -9,8 +9,54 @@ import OrientationUtils from 'Utils/OrientationUtils';
 import Coordinates from 'Core/Geographic/Coordinates';
 
 const coord = new Coordinates('EPSG:4326', 0, 0, 0);
+
+class FeatureContext {
+    #worldCoord = new Coordinates('EPSG:4326', 0, 0, 0);
+    #localCoordinates = new Coordinates('EPSG:4326', 0, 0, 0);
+    #geometry = {};
+
+    constructor() {
+        this.globals = {};
```

Comment:
I'm not fond of this `globals` property; I haven't found a better way to do it yet but we can at least document it